jentool/__init__.py

```diff
@@ -103,6 +103,11 @@ def _parser():
     parser_jobs_failing.add_argument('pattern', metavar='pattern', help='the Jenkins job name(s) (regex)')
     parser_jobs_failing.add_argument('--max-score', '-m', default=0, type=int, help='the maximum health score to look for')
     parser_jobs_failing.set_defaults(func=jobs_failing)
+    # jobs failing by view
+    parser_jobs_failing_by_view = subparsers.add_parser(
```

Comment:
could that be included in `jobs-failing` eg. with `jentool jobs-failing --use-view my-view-name` ?

Reply:
So it's not that simple because `get_jobs` and `get_jobs_info_regex` do not take the same parameters nor return the same values. For example, `get_jobs_info_regex` doesn't let you filter by views and it returns the `score` in the info dict while `get_jobs` doesn't.